iphone_connector.py

```diff
@@ -119,11 +119,33 @@ def initialize():
     # of the files within them.
     has_subdirectory_structure = MESSAGE_DB[:2] in os.listdir(_latest_sync_dir)
     if has_subdirectory_structure:
-        _message_con = sqlite3.connect(os.path.join(_latest_sync_dir, MESSAGE_DB[:2], MESSAGE_DB))
-        _address_con = sqlite3.connect(os.path.join(_latest_sync_dir, ADDRESS_DB[:2], ADDRESS_DB))
+        message_path = os.path.join(_latest_sync_dir, MESSAGE_DB[:2], MESSAGE_DB)
+        address_path = os.path.join(_latest_sync_dir, ADDRESS_DB[:2], ADDRESS_DB)
     else:
-        _message_con = sqlite3.connect(os.path.join(_latest_sync_dir, MESSAGE_DB))
-        _address_con = sqlite3.connect(os.path.join(_latest_sync_dir, ADDRESS_DB))
+        message_path = os.path.join(_latest_sync_dir, MESSAGE_DB)
+        address_path = os.path.join(_latest_sync_dir, ADDRESS_DB)
+
+    _message_con = sqlite3.connect(message_path)
+    _address_con = sqlite3.connect(address_path)
+
+    # Try to read from the DB files as a check if they are encrypted and raise exception accordingly
```

Comment:
How about:

`# Try to read from the DB files as a check to see if they are encrypted.`